src/components/NewCommentForm.tsx

```diff
@@ -75,18 +156,26 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('input', { 'is-danger': isError.body })}
```

Comment:
The `className` for the textarea should be 'textarea' instead of 'input'. This is likely a typo and could affect the styling of the textarea element.